src/sagas/key/serviceFunctions.js

```diff
@@ -0,0 +1,27 @@
+import containerPromise from '../../services/container';
+import { compressMnemonic } from '../../utils/key';
+
+export async function createPrivateKey() {
```

Comment:
Why exactly did you create a "proxy" for the function's?

Reply:
It looks better for me to abstract sagas from container and conversion (like mnemonic array to string and other). I've extracted all conversion from sagas, so it may make more sense now.